src/util/embedTagHelpers.js

```diff
@@ -19,6 +19,26 @@ export const reduceElementDataAttributes = el => {
   return obj;
 };
 
+export const reduceChildElements = el => {
```

Comment:
Lage en test på denne?

Reply:
@chrpeter hvordan kan jeg teste et slate element?

Reply:
Noe lignende er vel gjort her: https://github.com/NDLANO/editorial-frontend/blob/master/src/util/__tests__/slateHelpers-test.js ?

Reply:
Fikk det til på et vis alikevel, testet hele deserialiseringen i stedet for selve funksjonen (var muligens det du mente)